skimage/exposure/exposure.py

```diff
@@ -114,9 +114,11 @@ def histogram(image, nbins=256, source_range='image', normalize=False):
     --------
     >>> from skimage import data, exposure, img_as_float
     >>> image = img_as_float(data.camera())
-    >>> np.histogram(image, bins=2)
+    >>> counts, bins = np.histogram(image, bins=2)
+    >>> counts.astype(np.int), bins
```

Comment:
I am really -1 on this. It is extremely distracting to the example. 

This is not a programming style that we want to encourage others using.